EKrut-Server/src/ekrut/server/db/OrderDAO.java

```diff
@@ -204,4 +205,30 @@ public boolean updateOrderStatus(int orderId, OrderStatus status) {
 		
 		return true;
 	}
+
+	
+	
+	// For Almog: Added by Nir, get list of orders in Shipment status.
+	public ArrayList<Order> fetchOrderShipmentList(String area){
+		PreparedStatement ps = con.getPreparedStatement("SELECT orderId FROM orders WHERE type = SHIPMENT, area = ?");
```

Comment:
This SQL query is wrong.
First, SHIPMENT is a string and therefore needs to be quoted: `type = 'SHIPMENT'`.
Second, the comma is wrong. If you have multiple conditions that all need to be true you use `AND`:
`type = 'SHIPMENT' AND area = ?`.

Reply:
Fixed